src/scipp/html/style.css.template

```diff
@@ -58,7 +58,9 @@
   color: var(--sc-font-color2);
 }
 
-.sc-sections {
+/* Also override settings of reveal.js as introduced when
+   generating slides using the rise package. */
```

Comment:
What is the origin of this issue actually? Do they happen to use the same variable names, or something else?

Reply:
No, they have styles for `.reveal ul` and `.reveal ul ul` which set `display` to something that messes up our HTML repr. And theirs takes precedence over ours even though they don't specify the class of the `ul` (i.e. `sc-...`).